seatunnel-connectors-v2/connector-doris/src/main/java/org/apache/seatunnel/connectors/doris/sink/writer/DorisSinkWriter.java

```diff
@@ -255,12 +253,10 @@ public void close() throws IOException {
 
     @VisibleForTesting
```

Comment:
Can we remove this comment because it doesn't look right

Reply:
You are right. Removed